sale_measure/__openerp__.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) All Rights Reserved 2015 Akretion
+#    @author Abdessamad HILALI <abdessamad.hilali@akretion.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+{
+    'name': "Sale measure",
+    'summary': """Manage trainings""",
+    'description': """
+        Sale measure
+    """,
+    'author': "Akretion",
+    'website': "http://www.Akretion.com",
+    'license': "AGPL-V3",
+    'category': 'sale',
+    'version': '0.1',
+    'depends': ['base', 'sale', 'product'],
```

Comment:
You do not need to depend of base and product as sale already depend of this module